static/js/settings_org.js

```diff
@@ -412,6 +417,8 @@ function discard_property_element_changes(elem) {
         elem.prop('checked', property_value);
     } else if (typeof property_value === 'string' || typeof property_value === 'number') {
         elem.val(property_value);
+    } else if (typeof property_value === 'object') {
+        exports.populate_auth_methods(property_value);
```

Comment:
While splitting makes sure to have an `if` condition for this property or comment like you wrote below in  `check_property_changed`.